src/xapiwrapper.js

```diff
@@ -1,3 +1,4 @@
+(function(ADL, root){
 // adds toISOString to date objects if not there
 // from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString
```

Comment:
I've brought all of these functions into the closure defining this module. This is intentional, there doesn't seem to be a good reason to leak these functions into the global scope.
I would add that shimming the native Date object isn't a good idea as this library is clearly intended for use in other, potentially conflicting systems.
Changing it to the following might be more appropriate:
```js
function DateToISOString(date) {

}
```